api/room_resource/schema.py

```diff
@@ -290,6 +297,21 @@ def resolve_get_resources_by_room_id(self, info, room_id):
             resources.append(room_resource)
         return RoomResources(roomResources=resources)
 
+    @Auth.user_roles('Admin')
+    def resolve_resource_by_name(self, info, search_name):
+        resource = []
+        resource_name = search_name.lower().replace(" ", "")
```

Comment:
@mnswaleh 
This line suggests that I cannot search for resources that are space separated. Can you throw more light as to why this decision?

Reply:
The user can put space or not but when I search I remove the spaces from both the user input and the name in the database to increase chances for getting a match. For example, if a user searches for `tableone` and in the database it is saved as `table one` or vise versa